src/Lykke.Service.PayVolatility/Program.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Lykke.Sdk;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Lykke.Service.PayVolatility
+{
+    internal sealed class Program
+    {
+        public static string EnvInfo => Environment.GetEnvironmentVariable("ENV_INFO");
+
+        public static async Task Main(string[] args)
+        {
+            Console.WriteLine($"{PlatformServices.Default.Application.ApplicationName} version {PlatformServices.Default.Application.ApplicationVersion}");
+#if DEBUG
+            Console.WriteLine("Is DEBUG");
+#else
+            Console.WriteLine("Is RELEASE");
+#endif                       
+            try
+            {
+                var host = new WebHostBuilder()
+                    .UseKestrel()
+                    .UseUrls("http://*:5000")
```

Comment:
it is better to remove this line to be able to configure port locally.

Reply:
I agree with you, but this is company approach from template: https://github.com/LykkeCity/Lykke.Sdk/blob/bafc70a8cc9f6e7afdcc59205eacc4fe50344eed/src/Lykke.Sdk/LykkeStarter.cs#L45

I think devops use docker configuaration  for manage hosting url and ports.